webhooks/2025-04-15.yml

```diff
@@ -2551,172 +2563,111 @@ components:
-          type: boolean
-          description: Whether this is the default plan for payees
+          description: Display name of the plan
         is_recurring:
           type: boolean
           description: Whether this plan charges on a recurring basis
-        is_prorated:
-          type: boolean
-          description: Whether charges for this plan are prorated when changes occur mid-cycle
         amount:
           type: integer
           format: int64
-          description: Monthly billing amount in the smallest currency unit (actual charge may differ for annual plans)
-        period:
-          type: string
-          description: Billing period for this plan (e.g., "month", "annual")
-        interval:
-          type: integer
-          description: Number of period units between each billing cycle (e.g., 1 for monthly, 12 for yearly)
-        has_base_fee:
-          type: boolean
-          description: Whether this plan includes a base fee in addition to usage-based charges
+          description: Monthly billing amount in the smallest denomination of the currency (actual charge may differ for annual plans)
         currency:
           type: string
           description: Three-letter ISO 4217 currency code for this plan's pricing
-        annual_monthly_amount:
-          type: integer
-          format: int64
-          description: Effective monthly amount when billed annually, accounting for any annual discount
-        publicly_visible:
-          type: boolean
-          description: Whether this plan is visible to customers in public pricing pages
       required:
         - id
-        - instance_id
-        - product_id
-        - name
         - slug
-        - is_default
+        - name
         - is_recurring
-        - is_prorated
         - amount
-        - period
-        - interval
-        - has_base_fee
         - currency
-        - annual_monthly_amount
-        - publicly_visible
     BillingSubscriptionItem:
       type: object
       properties:
         id:
           type: string
           description: Unique identifier for the subscription item
-        status:
-          type: string
-          description: Current status of the subscription item (e.g., "active", "canceled", "past_due", "incomplete")
-        credit:
-          $ref: '#/components/schemas/BillingSubscriptionCredit'
-          description: Available credit information for the subscription item
-        proration_date:
+        object:
           type: string
-          description: Date used for proration calculations when subscription changes occur
-        plan_period:
+          description: String representing the object's type
+          enum:
+            - commerce_subscription_item
+        status:
           type: string
-          description: Billing period for the subscription item (e.g., "month", "annual")
-        period_start:
+          description: Current status of the subscription item
+          enum:
```

Comment:
Should we create a schema for this attribute to provide details as to what each `enum` `status` type is?

Reply:
Huh, I was unaware that's a thing we could do, if I'm being honest.

I'm not against it, do you have an example offhand of how it would be achieved?

Reply:
Apologies, I thought it was apart of OAS, but it's not. In this case, can we add descriptions to the `description` of `status`?